### DISCUSSION TITLE
Preview mode?

### DISCUSSION THREAD

**Discussion by Fierce-Ninja** [athensresearch/athens#1400] [General]
Is there some kind of preview mode when you are using markdown on the main note?  Like in obsidian there is an icon or you can do ctrl-click to see the preview of what you are writing.  Sorry if I overlooked this feature.  Just started using Athens.  Please advise.

**Comment by sawhney17** [comment]
Athens has a semi WYSIWG format. In short, when you click out of a block, you see the rendered version. Check out this video for an example of it in action. [https://www.loom.com/share/ad80734e3b2a459b8ac6dd258d134d9e?from_recorder=1&focus_title=1](https://www.loom.com/share/ad80734e3b2a459b8ac6dd258d134d9e?from_recorder=1&focus_title=1)

**Reply by Fierce-Ninja** [reply]
When I use [[ ]], the brackets still continue to show even when I click out of the block.

**Reply by sawhney17** [reply]
> When I use [[ ]], the brackets still continue to show even when I click out of the block.

That's actually intentional behavior. Even obsidian shares the same behavior in their preview mode. It's meant to be use to be able to navigate.
